scripts/lynx/upgrade_coordinator.py

```diff
@@ -18,12 +19,24 @@ def main():
     deployer = Deployer.from_yaml(filepath=CONSTRUCTOR_PARAMS_FILEPATH, verify=VERIFY)
     instances = contracts_from_registry(filepath=ARTIFACTS_DIR / "lynx.json", chain_id=80002)
 
-    coordinator = deployer.upgrade(
-        project.Coordinator, instances[project.Coordinator.contract_type.name].address
+    implementation = deployer.deploy(project.Coordinator)
+    # `initializeNumberOfRituals` was used for the original upgrade, but should NOT be
+    #  used for subsequent upgrades of Coordinator
+    # encoded_initializer_function = implementation.initializeNumberOfRituals.encode_input()
+    encoded_initializer_function = b""
+    coordinator = deployer.upgradeTo(
```

Comment:
Can you just still use `upgrade()`, not deploying the implementation explicitly above?

Reply:
✔️